mava/wrappers/smac.py

```diff
@@ -46,10 +46,11 @@ def __init__(
             return_state_info: return extra state info
         """
         self._environment = environment
+
         self._return_state_info = return_state_info
         self._agents = [f"agent_{n}" for n in range(self._environment.n_agents)]
 
-        self._reset_next_step = True
+        self._is_resetted = False
```

Comment:
Minor grammatical thing, just rename to `_is_reset`, maybe even `_is_first_step` is a more descriptive, although not so sure about that one. 

I would also put a comment there explaining why we now do this, something like "because smac returns weird values if you reset twice in a row".